modules/react/layout/lib/Box.tsx

```diff
@@ -1,7 +1,10 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import {jsx} from '@emotion/react';
```

Comment:
I'm not sure this is the way we want to go about this or not. I'm open to suggestions here. We could also try to make this a style function similar to the other style props.

Reply:
I think when we were talking about this this is what would work, have the pragma in one file where we use the css prop and that's it, im ok with this!